plugin.video.gamekings/addon.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python
+# -*- coding: UTF-8 -*-
+
+#
+# Imports
+#
+from future import standard_library
+standard_library.install_aliases()
+from builtins import str
+import os
+import sys
+import urllib.parse
+import xbmc
+import xbmcaddon
+
+LIB_DIR = xbmc.translatePath(
+    os.path.join(xbmcaddon.Addon().getAddonInfo('path'), 'resources', 'lib'))
+sys.path.append(LIB_DIR)
+
+from gamekings_const import ADDON, SETTINGS, DATE, VERSION
+
+# Parse parameters
+if len(sys.argv[2]) == 0:
+    #
+    # Main menu
+    #
+    xbmc.log("[ADDON] %s, Python Version %s" % (ADDON, str(sys.version)), xbmc.LOGDEBUG)
+    xbmc.log("[ADDON] %s v%s (%s) is starting, ARGV = %s" % (ADDON, VERSION, DATE, repr(sys.argv)),
+                 xbmc.LOGDEBUG)
+
+
+    if SETTINGS.getSetting('onlyshowvideoscategory') == 'true':
```

Comment:
```suggestion
    if SETTINGS.getSettingBool('onlyshowvideoscategory'):
```

Reply:
Fixed. I'll make a separate pr to set the addon to broken.